.gitignore

```diff
@@ -1 +1,3 @@
 dist
+.vscode
+.env
```

Comment:
This file should only include items that are generated by code in this repo.

The above two look IDE specific. Instead, I recommend considering setting up a global ignore on your machine. That would then apply to every git repo cloned on your machine, and would avoid repeating this in every git repo you clone.

Reply:
hi Masih,
This is not the one that jamshid worked on. It is a new one and only the coeds that we wrote ourselves